docusaurus/docs/introduction/plugin-types-usage

```diff
@@ -0,0 +1,133 @@
+
+### Bundling of dashboards
+
+Data-source plugins can include dashboards by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a data source plugin in the `General` folder when it is imported.
+
+## App plugins
+
+Development of app plugins, also known as apps or applications, allows you to bundle data sources and panels to provide a seamless monitoring experience, similar to existing plugins such as [Zabbix](https://grafana.com/grafana/plugins/alexanderzobnin-zabbix-app/).
+
+### Usage of apps
+
+You can configure a Grafana app when you want to integrate with a third-party service and optionally provide custom pages and such within Grafana. Given any installed plugin of type `app`, you can enable them once per per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations) and they are persisted in Grafana’s database. 
+
+Use the plugin settings API [`/api/plugins`](https://grafana.com/api/plugins), `/api/plugins/\<plugin id\>/settings/`) for endpoints to read and update app settings. For example, you can enable or disable the app with your preferred configurations. 
+
+:::note
+
+To distinguish a Grafana app from an app plugin, we sometimes refer to the latter as an _app instance_, that is, a configured Grafana app with a `plugin id` type of `app`. 
+
+::: 
+
+### Provisioning of apps
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) to configure your app plugins.
+
+Apps can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/) so that you can use YAML files to start them when Grafana starts or on demand. Note that the plugin must be installed before provisioning can succeed with a `plugin id` of `app` type.
+
+### Bundling of apps
+
+The app plugin type allows you to [nest other plugins inside it](../create-a-plugin/extend-a-plugin/nested-plugins); in other words, to bundle or distribute multiple plugins in the same package. 
+
+### Bundling of dashboards
+
+App plugins can include dashboards by referencing dashboard JSON files within the `plugin.json` including `property` and `type=dashboard`. Grafana puts an app plugin in the `General` folder when it is imported.
+
+## Panel (visualization) plugins
+
+Development of panel plugins, also known as visualizations, allows Grafana to use custom visualizations, similar to existing plugins such the [Worldmap](https://grafana.com/grafana/plugins/grafana-worldmap-panel/) panel. 
+
+### Usage of panel plugins
+
+Any installed plugin of type `panel` can be used, selected, and configured as a visualization within a dashboard.
+
+Use the plugins API ([`/api/plugins`](https://grafana.com/api/plugins), `/api/plugins/\<plugin id\>/`) for endpoints to retrieve installed plugins, including panel plugins. 
```

Comment:
I am not sure if they need this - they can always check the list of installed plugins in the catalog (until we have an actual API-documentation). What do you think?